source/frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -261,28 +261,20 @@ export const toCqlFilter = (
  * NOTE: The resulting string value is left as-is. It will not be URL-encoded
  * @param object an object to convert to a cql filter string.
  */
-export const toCqlFilterValue = (
-  object: Record<string, string>,
-  forceSimplePid?: boolean,
-  forceExactMatch?: boolean,
-) => {
+export const toCqlFilterValue = (object: Record<string, string>, flags?: IWfsCqlFlags) => {
```

Comment:
standardized the cql flags in an interface that limits the risk of flags being sent in the wrong order.